sources/packages/web/src/services/http/UserApi.ts

```diff
@@ -40,4 +44,17 @@ export class UserApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async allUserLocationsApi(headers?: any): Promise<UserLocationDto[]> {
```

Comment:
This method does not need to support the custom header parameter.
The methods that have that are methods that need to be called too early in the process when the authentication token is not even in place and accessible globally.

Reply:
Done